WcaOnRails/app/models/competition.rb

```diff
@@ -515,8 +521,12 @@ def has_entry_fee?
     base_entry_fee.nonzero?
   end
 
+  def has_competitor_limit?
+    competitor_limit_flag
```

Comment:
Ah! My bad. If we have a `competitor_limit_flag` column, then `competitor_limit` should definitely not be nullable. I'm equally supportive of these 2 options:

1. Rename `competitor_limit_flag` to `competitor_limit_enabled`. Do not allow null values for `competitor_limit_enabled` *or* `competitor_limit`.
2. Remove `competitor_limit_flag` field and change NULL `competitor_limit` to mean "this competition does not enforce a competitor limit".

In either situation, we should not change the UI (I'm pleased with the current one).